lib/utils.js

```diff
@@ -0,0 +1,11 @@
+var path = require('path');
+
+/* Normalize URL or path to an absolute path, if it corresponds to one. */
+function filePath(urlOrPath) {
```

Comment:
Good point. I'll go back and look at where "absolute relative" paths should be sanitized